README.md

```diff
@@ -33,40 +31,55 @@ Available modules
 * `status`
 * `task`
 
-Installation
----
+## Installation
 
-After building the collection artifact with `make dist`, you can install the resulting `tar.gz` file with `ansible-galaxy` in the usual ways.
+### Install from Ansible Galaxy
 
-Documentation
----
+The collection is available from Ansible Galaxy, so you can install it via
 
-You can find the inline documentation of each module with `ansible_doc pulp.squeezer.pulp_<...>`.
+    $ ansible-galaxy collection install pulp.squeezer
 
-Testing
----
+### Build locally
+
+Alternatively you can building the collection artifact with
+
+    $ make dist
+
+and install the resulting `tar.gz` file with
+
+    $ ansible-galaxy collection install pulp-squeezer-<version>.tar.gz
+
+## Documentation
+
+You can find the inline documentation of each module with `ansible-doc pulp.squeezer.<module_name>`.
+
+## Testing
 
 Testing is done by running handcrafted playbooks from `tests/playbooks` while playing back prerecorded server answers.
 Using python virtual environments is recommended.
 
-Those playbooks are meant to test one specific module, and are usually structured in three sections:
+There is usually one playbook per module that it is meant to test, but that is not a hard requirement.
+
+The playbooks are usually organized in a three-act structure:
 
- * In the first part, ran against `localhost`, fixtures can be set up.
- * The second part should contain the actual tests and is executed on the virtual host `tests` to allow recording.
- This usually involves calling the module in question several times with varying parameters and verifying it's output.
- * The third part, again for `localhost`, provides the the opportunity to undo part one.
+ * In the first act, the setup, ran against `localhost`, fixtures can be prepared.
+ * The second act, the confrontation, should contain the actual tests.
+   It is executed on the virtual host `tests` to allow recording the requests to the rest api with the corresponding responses.
+   This usually involves calling the module in question several times with varying parameters and verifying it's output.
+ * The third act, the resolution, again with the target `localhost`, provides the the opportunity to undo part one.
 
-In the first and third parts, no recording takes place.
-During playback, only the part in the middle is executed.
+In the first and third acts, no recording takes place.
+During playback, only the act in the middle is executed.
+Please make sure, that this act can run independently from the others.
```

Comment:
This is more a question rather than a suggestion, but are there any specific steps they can do to ensure this? Or is it just something that someone using this would know?

Reply:
This is more like things not to do. e.g. not depending on variables that are set in the first part, because it will not be executed in the playback case.